File: yl10.lua

io.write "Mis on teie nimi? "
local name = io.read("*line")

print(("Tervist, %s!"):format(name))

io.write "Kus te elate? "
local place = io.read("*line"):lower

print(
	place:find "saarema" and "Kohalik, või?"
	or "Kõlab nagu tore koht :)"
)

io.write "Kui vana olete? "
local age = io.read("*number")

print(
	age < 18 and "Olete liiga noor, et autot juhtida :/"
	or age == 18 and "Palju õnne täisealiseks saamise puhul!"
	or "Olete ilmselt autojuhtimiseks õiges eas."
)

File: yl10.py

name = input("Mis on teie nimi?")
print(f"tervist, {name}!")

place = input("Kus te elate?").lower()
if place.find("saaremaa") != -1:
    print("Kohalik, või?")
else:
    print("Kõlab nagu tore koht :)")

age = int(input("Kui vana olete?"))

if age < 18:
    print("Olete liiga noor, et autot juhtida :/")
elif age == 18:
    print("Palju õnne täisealiseks saamise puhul!")
else:
    print("Olete ilmselt autojuhtimiseks õiges eas.")
